common/src/enums/permissions.ts

```diff
@@ -1,5 +1,4 @@
 export enum Permissions {
-    AccessBusinessPortal = 0,
```

Comment:
We should leave this in commented out or deprecated as below (my vote is commented out if there are not upgrade concerns)

We can't reuse values in this enum, so it's best to leave reminders to what permission values were and what are available.

Reply:
I believe the enum is only used internally in the projects? So it should be possible to change the enum values.